src/special/package.js

```diff
@@ -0,0 +1,30 @@
+export default function parsePackageJson(content, filepath, deps) {
+  // Mark dependencies as "used" if they are used in a script in package.json.
+  if (filepath.match(new RegExp('(\\/|\\\\)package.json$'))) {  // Escaping is fun!
+    // Grab just the scripts section.
+    const scripts = JSON.parse(content).scripts;
+
+    if (scripts) {
+      const scriptKeys = Object.keys(scripts);
+
+      if (scriptKeys.length) {
+        // Don't count script titles as dependency use.
+        let scriptsText = '';
+        for (let i = 0; i < scriptKeys.length; i += 1) {
+          scriptsText += `${scripts[scriptKeys[i]]} `;
+        }
+
+        if (scriptsText) {
+          const depsInScripts = [];
+          for (let i = 0; i < deps.length; i += 1) {
+            if (scriptsText.indexOf(deps[i]) !== -1) {
```

Comment:
This may not work like that. 😢 

The script contains the dependency string does not *really* means it is using that dependency. 

---

BTW, even if compare string is acceptable, it needs to [grab the `bin` section](https://github.com/depcheck/depcheck/blob/master/src/special/bin.js#L41) from `dep/package.json` for comparison.

Reply:
btw. the `bin` field can be both a string or an object. When it's a string, the name of the binary is the same as the name of the package. When it's an object, every key corresponds to the binary name.

Reply:
@LinusU Good catch! Could you please fix the string case in the `bin` special? Thank you!